nncf/experimental/onnx/graph/onnx_graph.py

```diff
@@ -285,9 +285,9 @@ def get_edge_shape(self, edge_name: str) -> List[int]:
         self._update_activation_tensors(do_shape_inference=True)
         if edge_name in self._activations_tensor_name_to_value_info:
             return ONNXGraph.get_tensor_shape(self._activations_tensor_name_to_value_info[edge_name])
-        raise RuntimeError('There is no edge with the name {}'.format(edge_name))
+        return None
```

Comment:
Why it returns `None` now? Is it checked somehow in the code that uses this method? Can't we get unpredictable behavior during quantization?

Reply:
I revert these changes, we can discuss later about returning None or raising the Exception during refactoring ONNXGraph